datasource/get_watertemp.go

```diff
@@ -77,3 +77,39 @@ func WaterTemperatureSchemaToString(wts WaterTemperatureSchema) string {
 func hasLowerTemperature(location1 WaterTemperatureSchema, location2 WaterTemperatureSchema) bool {
 	return location1.Temp_water < location2.Temp_water
 }
+
+/* TO DO: Need to discuss this part of the code!
+The task was to "Create function that drops data from watertemp schema and converts the data
+to a json object (name of location and temperature) - create own class"
+Is it a reasonable solution?
+Is it correctly located or should for example ColdSiteStruct be located elsewhere? */
+
+// Define a struct type ColdSiteStruct
+type ColdSiteStruct struct {
```

Comment:
Move this struct into a new datasource/api.go. Also move  WaterTemperatureSchema there